deploy/tests/frontmatter.rb

```diff
@@ -0,0 +1,74 @@
+require 'yaml'
+def loadconfig()
+	config = YAML.load_file('_config.yml')
+end
+@config = loadconfig
```

Comment:
Code style note, this is (IMO) a cleaner way of expressing this:

```ruby
def config
	@config ||= YAML.load_file '_config.yml'
end
```

And then subsequent references can be made to `config` instead of `@config`.

This gives you the same effect -- the config file is only read in once, and is cached in memory for subsequent calls. So you can continue to reference `config`, and it hardly matters whether it's a method or variable.

I also removed the empty parens, which in most Ruby code I've seen/written aren't used for methods which take no parameters.